.github/workflows/autodocs-platform.yaml

```diff
@@ -69,26 +69,23 @@ jobs:
         egress-policy: audit
 
     - name: 'Checkout default branch to $GITHUB_WORKSPACE dir'
-      uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3
+      uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
 
     - name: 'Setup gitsign'
       uses: chainguard-dev/actions/setup-gitsign@main
 
     - name: Authenticate to Google Cloud
       id: auth
-      uses: google-github-actions/auth@ceee102ec2387dd9e844e01b530ccd4ec87ce955 # v0
+      uses: google-github-actions/auth@71fee32a0bb7e97b4d33d548e7d957010649d8fa # v2.1.3
       with:
-        token_format: 'access_token'
         project_id: "${{ secrets.PROJECT_ID }}"
         workload_identity_provider: "${{ secrets.GCP_WORKLOAD_IDENTITY_PROVIDER }}"
```

Comment:
you still need to have the svcc account, see this as example: https://github.com/chainguard-dev/infra-images/blob/main/.github/workflows/deploy-staging-images-bundle.yaml#L29-L33